### PR TITLE
Associate management

### PR DESCRIPTION
**Pull request by BohdanAndrusiak**

### CODE REVIEW

File: src/main/java/com/ita/if103java/ims/service/impl/AssociateServiceImpl.java [hunk 1]
```diff
@@ -70,8 +74,8 @@ public AssociateDto update(UserDetailsImpl user, AssociateDto associateDto) {
     }
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/service/impl/AssociateServiceImpl.java]: Just mentioned that address is not updated when update associate.
Please add here update of the address and also mark this method as transactional

**Comment by ikrasWork** [comment]
Reviewed last commit. good

File: src/main/java/com/ita/if103java/ims/service/impl/WarehouseItemAdviceServiceImpl.java [hunk 2]
```diff
@@ -22,6 +22,7 @@
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.function.Function;
```
Comment (rostIvan) [inline src/main/java/com/ita/if103java/ims/service/impl/WarehouseItemAdviceServiceImpl.java]: Remove it, pls

Comment (rostIvan) [reply src/main/java/com/ita/if103java/ims/service/impl/WarehouseItemAdviceServiceImpl.java]: Can be, in any case we'll fix all the imports, indention tomorrow

**Pull request merged**